pkg/clui/nop.go

```diff
@@ -0,0 +1,12 @@
+package clui
+
+var Nop = &nopImpl{}
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported var Nop should have comment or be unexported